src/components/screens/forms/forms-data-screen.tsx

```diff
@@ -96,7 +96,9 @@ const FormsDataScreen: React.FC = () => {
    */
   const loadAuditLogEntries = async (metaformId: string): Promise<AuditLogEntry[]> => {
     try {
-      return await auditLogEntriesApi.listAuditLogEntries({ metaformId: metaformId });
+      const auditLogEntries = await auditLogEntriesApi.listAuditLogEntries({ metaformId: metaformId });
+      
+      return auditLogEntries;
```

Comment:
What's the point of this?

Reply:
Linter was giving errors about, if I remember correctly, no-return-await rule.